notebooks/jupyter_tutorial.md

```diff
@@ -0,0 +1,37 @@
+# Tutorial: Running a Jupyter Lab Workload
+A workload is the actual job or task you want to run on the platform. This could be training an AI model, running inference, doing data preprocessing, or conducting a scientific simulation.
+
+Generally, the minimum requirements you need before creating the workload include:
+
+* Being granted permission to an active project
+* An environment to run such job
```

Comment:
Maybe a link to the environments section so peaople know what you mean by environment.